### ISSUE TITLE
[BUG]: Street Mahjong Trans-Asakami 2 (Japan) [SLPS-20017] [CRC32-e5b63878]

### ISSUE THREAD

**Issue by spacemul** [PCSX2/pcsx2#5121]
### Describe the Bug

Black screen after Sunsoft logo

![gs_20211207215655](https://user-images.githubusercontent.com/7451522/145239695-beae83aa-2b13-463e-a1af-a4ec62f647e6.png)

### Reproduction Steps

Just launch the game

### PCSX2 Revision

1.7.2103

### Operating System

Windows 10 (64bit)

### CPU

Ryzen 5 3600

### GPU

RTX 3070

**Comment by gilderoylockhart01** [comment]
I sugest you try the interpreters and especially the EE with the cache enabled.

After that you can also try the various fixes/hacks in the hack section but only one by one.

**Comment by refractionpcsx2** [comment]
This is waiting to read the VSync INTC flag (by the looks of things) but probably getting eaten too quickly.

**Comment by gilderoylockhart01** [comment]
Is this altered by our intc speed hack somehow?

**Comment by prafullpcsx2** [comment]
Tracked it down to https://github.com/PCSX2/pcsx2/commit/4524fd919e2cc91eb814914c3c51b15d891d027a

**Comment by refractionpcsx2** [comment]
Makes sense, that does alter the behaviour of intc irq's being raised, but our system isn't really designed to allow what the game is trying to do. Penny Racers (Gadget Racers) have the same issue, apparently Grandia 2 does simiar?

**Comment by Goatman13** [comment]
I made patch for it, terrible hack but seems to work ok.
Patch is little bit extensive because IOP offset is not guaranteed to be the same on different BIOS (reloc).
```
// Hack sceCdDiskReady to not wait for drive ready.
patch=1,EE,000ffef4,word,304200c0
patch=1,EE,000ffef8,word,1443fffb
patch=1,EE,000fff00,word,3c19bc01
patch=1,EE,000fff04,word,9f2f0000
patch=1,EE,000fff08,word,3c180010
patch=1,EE,000fff0c,word,9f18fef4
patch=1,EE,000fff10,word,130f0003
patch=1,EE,000fff14,word,27390004
patch=1,EE,000fff18,word,1000fffa
patch=1,EE,000fff1c,word,00000000
patch=1,EE,000fff20,word,9f2f0000
patch=1,EE,000fff24,word,3c180010
patch=1,EE,000fff28,word,9f18fef8
patch=1,EE,000fff2c,word,170ffff6
patch=1,EE,000fff34,word,af200000
patch=1,EE,000fff38,word,03e00008
patch=1,EE,00273870,word,0803ffc0
```
